priv/templates/list/list-item.frequently_asked_question.tpl

```diff
@@ -10,21 +10,11 @@
             <article>
                 <div class="list__item__content--faq">
                     <div class="list__item__title list__item__short-title u-margin-none">
-                        <h3>
-                            {% if id.short_title %}
-                                {{ id.short_title|truncate:40 }}
-                            {% else %}
-                                {{ id.title|truncate:40 }}
-                            {% endif %}
-                        </h3>
+                        <h3>{{ id.title }}</h3>
```

Comment:
Do we not want to truncate this at all? Does it still look good with a very long title?

Reply:
Yup looks good, title will just wrap fine